userspace/libsinsp/utils.cpp

```diff
@@ -1537,6 +1537,76 @@ bool sinsp_utils::startswith(const std::string& s, const std::string& prefix)
 	return strncmp(s.c_str(), prefix.c_str(), prefix_len) == 0;
 }
 
+const vector<string> capabilities {
+	{"CAP_CHOWN"},
```

Comment:
To retain portability, we generally define ppm-specific values whenever interpreting binary values as strings, even when the binary values have well known mappings. For example, https://github.com/falcosecurity/libs/blob/master/driver/ppm_events_public.h#L43 and https://github.com/falcosecurity/libs/blob/master/driver/ppm_flag_helpers.h#L300.

I think you want to do the same thing here.

Reply:
Thanks!